keras_core/random/seed_generator.py

```diff
@@ -103,49 +106,9 @@ def global_rng_state():
 
     Returns:
         A `KerasVariable` with shape `(2,)` and dtype `uint32`.
-
-    This is the global state used by unseeded (`seed=None`) Keras
-    random ops, e.g. `keras_core.random.normal(shape=(), seed=None)`.
-
-    In JAX, if you're using unseeded random ops, be mindful that
-    their outputs will be unchanged across different calls of
-    a traced function (e.g. a `jax.jit`-transformed function) since
-    traced functions in JAX are fully stateless. To get
-    different outputs across different calls, you will need to pass the
-    global RNG state in and out of the function boundary, like this:
-
-    ```python
-    @jax.jit
-    def random_numbers(seed):
-        rng_state = keras_core.random.global_rng_state()
-        rng_state.assign(seed)
-        x = keras_core.random.normal((), seed=None)
-        y = keras_core.random.normal((), seed=None)
-        return x, y, rng_state.value
-
-    seed = jax.numpy.array([0, 0])
-    x, y, seed = random_numbers(seed)
-    new_x, new_y, seed = random_numbers(seed)
-    ```
     """
-    return global_seed_generator().state
 
-
-def make_default_seed():
-    return python_random.randint(1, int(1e9))
```

Comment:
This is the code we should use for the default seed instead of `42`

Reply:
Yes, I am aware of this. I will make that change in the final pass when I rewrite tests